src/yaml/saasquatch-api.yaml

```diff
@@ -468,6 +468,34 @@ paths:
                 description: "The `id` of the Account associated with the User being tracked. See [Shared vs. Solo accounts](/shared-vs-solo-accounts/) for more info."
                 type: string
                 example: "someAccountId"
+              idempotencyKey:
+                description: |
+                  The idempotency key for a logUserEvent request.
```

Comment:
A lot of this detail maybe should actually live in the endpoint description? We definitely should mention idempotency to some extent in the endpoint description. Also it just looks really big in the docs 
![image](https://user-images.githubusercontent.com/50340998/190255742-18b327e3-bf3a-4792-b1e3-562a47718d8e.png)